src/monitoring_service/handlers.py

```diff
@@ -41,16 +41,23 @@
 class Context:
     ms_state: MonitoringServiceState
     db: Database
-    w3: Web3
+    web3: Web3
     monitoring_service_contract: Contract
     user_deposit_contract: Contract
     min_reward: int
     required_confirmations: int
 
     @property
-    def latest_known_block(self) -> BlockNumber:
+    def latest_commited_block(self) -> BlockNumber:
```

Comment:
Should we have both names `latest_commited_block` and `latest_known_block` for the same thing?

Reply:
Good point, will change in `BlockchainState` as well